argon2.gemspec

```diff
@@ -11,6 +11,8 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Technion"]
   spec.email         = ["technion@lolware.net"]
 
+  spec.required_ruby_version = '>= 2.6.0'
```

Comment:
Per https://github.com/technion/ruby-argon2/commit/411016852bde0976ff2bb8484b3fc2c82edd2a0b, Ruby 2.5 and below are not supported.